protocol/serialization-types/tests/test_deserialize_staged_ledger_pre_diff_one.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use mina_serialization_types::v1::StagedLedgerPreDiffOneV1;
+use test_fixtures::TEST_BLOCKS;
```

Comment:
Same with this one. Lets keep them all standard

Reply:
removed

Reply:
removed